content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,41 +83,176 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
 
 .. tip::
    Reordering rules can also be created from each product form. To do so, navigate to
-   :menuselection:`Inventory --> Products --> Products`, then select a product. Click on
-   :menuselection:`Reordering Rules --> Create`, then fill out the new line as detailed above.
+   :menuselection:`Inventory app --> Products --> Products`, then select a product. Click on
+   :menuselection:`Reordering Rules --> New`, then fill out the new line, as detailed above.
+
+For advanced usage of reordering rules, learn about the following reordering rule fields:
+
+- :ref:`Trigger <inventory/product_management/trigger>`
+- :ref:`Visibility days <inventory/product_management/visibility-days>`
+- :ref:`Preferred route <inventory/product_management/route>`
+
+.. _inventory/product_management/trigger:
+
+Trigger
+=======
+
+When stock falls below the reordering rule's minimum, set the reordering rule's *trigger* to
+*automatic* to automatically create purchase or manufacturing orders to replenish stock.
+
+Alternatively, setting the reordering rule's trigger to *manual* displays the product and forecasted
+stock on the *replenishment dashboard*, where the procurement manager can review the stock levels,
+lead times, and forecasted dates of arrival.
+
+.. seealso::
+   :doc:`strategies`
+
+.. tip::
+   The replenishment dashboard is accessible by going to :menuselection:`Inventory app -->
```

Comment:
```suggestion
   The :guilabel:`Replenishment` dashboard is accessible by going to :menuselection:`Inventory app -->
```
I am suggesting the use of a GUI here because, once they use the provided directions to navigate to that specific page, the actual title of 'Replenishment' *does*, in fact, appear on the UI/screen at that time. So, that's my reasoning behind the suggestion. Feel free to take or leave (if you disagree with that logic) -- totally up to you 👍

Reply:
This logic makes sense to me-- thanks for the suggestion!